examples/http-server/send_image.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+import argparse
+import requests
+from typing import NamedTuple
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('hostname')
+    subparsers = parser.add_subparsers(dest='command')
+    clear_parser = subparsers.add_parser('clear')
+    draw_parser = subparsers.add_parser('draw')
+    draw_parser.add_argument('-c','--clear',action="store_true")
+    draw_parser.add_argument('file')
+    info_praser = subparsers.add_parser('info')
+
+    return parser.parse_args()
+
+def clear(hostname):
+    requests.post(f"http://{hostname}/clear").raise_for_status()
+
+class EpdInfo(NamedTuple):
+    width: int
+    height: int
+    temperature: int
+
+    @classmethod
+    def from_response(cls, resp):
+        return cls(
+            width=int(resp.headers['width']),
+            height=int(resp.headers['height']),
+            temperature=int(resp.headers['temperature']),
+        )
+
+class Dimensions(NamedTuple):
+    width: int
+    height: int
+
+def info(hostname):
+    resp = requests.get(f"http://{hostname}")
+    resp.raise_for_status()
+    return EpdInfo.from_response(resp)
+
+import PIL
```

Comment:
Just a nitpick: I'd put all the imports to the top.

Reply:
You're right